### PR TITLE
API Manager url and basic update

### PR DESCRIPTION
**Pull request by jeremiedmns**
Modification de l'url et de la clé suite à mise à jour

### CODE REVIEW

File: 3rdparty/cozytouch/client/CozyTouchApiClient.class.php [hunk 1]
```diff
@@ -45,12 +45,12 @@ public function authenticate()
 	public function getToken() {
 		$post_data = array(
 				'grant_type' => 'password',
-				'username' => $this->userId,
+				'username' => 'GA-PRIVATEPERSON/'.$this->userId,
```
Comment (iMicknl) [inline 3rdparty/cozytouch/client/CozyTouchApiClient.class.php:48]: @jeremiedmns could you explain what this `GA-PRIVATEPERSON/` addition means? I work on the CozyTouch (overkiz) integration for Home Assistant. :)

Comment (jeremiedmns) [reply 3rdparty/cozytouch/client/CozyTouchApiClient.class.php:48]: Hello @iMicknl 
GA-PRIVATEPERSON is the store of cozytouch accounts. If you don’t add it with new url, you could have troubles on login.
You can use same basic with new url for homeassistant integration.
Contact me if you have any question.

Comment (iMicknl) [reply 3rdparty/cozytouch/client/CozyTouchApiClient.class.php:48]: Thanks @jeremiedmns! I tested it without adding `GA-PRIVATEPERSON/`, and that seem to work. But I can imagine that adding this prefix will be more future ready.

Comment (iMicknl) [reply 3rdparty/cozytouch/client/CozyTouchApiClient.class.php:48]: @jeremiedmns where did you got this knowledge? (You can also reach out privately to me on Discord, iMick#1903).

I wonder if this is required, or only in scenarios where business accounts / private persons share the same email address.

**Pull request merged**